src/cli.js

```diff
@@ -71,9 +75,11 @@ function copyModelViewer(){
 
   const width = argv.width || 1024;
   const height = argv.height || 1024;
-  const format = argv.image_format || 'image/png';
   const quality = argv.image_quality || 0.92;
   const timeout = argv.timeout || 10000;
+  const [output, format] = scrubOutput(argv.output, argv.image_format)
+  const defaultBackgroundColor = (format === 'image/jpeg' ? 'rgba(255, 255, 255, 100)' : 'rgba(255, 255, 255, 0)');
```

Comment:
nit: I would save the `rgba(...)` in variables named after the colors. Makes it easier for readers